src/components/EditCaption/EditCaption.jsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
```

Comment:
Curious about your use of 'useLocation,' as I'm not familiar with this. Maybe you can teach me more about it in the context of your code!